crates/starknet-devnet-server/src/restrictive_methods.rs

```diff
@@ -0,0 +1,119 @@
+use std::collections::HashMap;
+
+use lazy_static::lazy_static;
+lazy_static! {
+    static ref MAPPING_HTTP_URIS_TO_RPC_METHODS: HashMap<&'static str, &'static str> =
+        HashMap::from([
+            ("/dump", "devnet_dump"),
+            ("/load", "devnet_load"),
+            ("/set_time", "devnet_setTime"),
+            ("/increase_time", "devnet_increaseTime"),
+            ("/create_block", "devnet_createBlock"),
+            ("/abort_blocks", "devnet_abortBlocks"),
+            ("/restart", "devnet_restart"),
+            ("/mint", "devnet_mint"),
+            ("/postman/load_l1_messaging_contract", "devnet_postmanLoad"),
+            ("/postman/flush", "devnet_postmanFlush"),
+            ("/postman/send_message_to_l2", "devnet_postmanSendMessageToL2"),
+            ("/postman/consume_message_from_l2", "devnet_postmanConsumeMessageFromL2"),
+            ("/predeployed_accounts", "devnet_getPredeployedAccounts"),
+            ("/account_balance", "devnet_getAccountBalance"),
+            ("/config", "devnet_getConfig"),
+        ]);
+    static ref MAPPING_RPC_METHODS_TO_HTTP_URIS: HashMap<&'static str, &'static str> =
```

Comment:
Why this Hungarian notation? The name could be simplifed without loss of information to something like `RPC_METHOD_TO_HTTP_URI` and vice versa.